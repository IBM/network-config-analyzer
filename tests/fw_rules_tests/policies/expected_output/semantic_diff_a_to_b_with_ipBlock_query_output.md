|query|src_ns|src_pods|dst_ns|dst_pods|connection|
|---|---|---|---|---|---|
|semantic_diff, config1: config_a_with_ipBlock, config2: config_b_with_ipBlock, key: Lost connections between removed peers||||||
||[default]|[app=app-3]|[default]|[app=app-4]|All connections|
||[default]|[app=app-4]|[default]|[app=app-3]|All connections|
|semantic_diff, config1: config_a_with_ipBlock, config2: config_b_with_ipBlock, key: Lost connections between removed peers and ipBlocks||||||
|||0.0.0.0/0|[default]|[app in (app-3,app-4)]|All connections|
||[default]|[app in (app-3,app-4)]||0.0.0.0/0|All connections|
|semantic_diff, config1: config_a_with_ipBlock, config2: config_b_with_ipBlock, key: Lost connections between removed peers and persistent peers||||||
||[default]|[app in (app-3,app-4)]|[default]|[app in (app-0,app-2)]|All connections|
||[default]|[app not in (app-3,app-4)]|[default]|[app in (app-3,app-4)]|All connections|
|semantic_diff, config1: config_a_with_ipBlock, config2: config_b_with_ipBlock, key: Added connections between persistent peers||||||
||[default]|[app=app-0]|[default]|[app=app-1]|All connections|
|semantic_diff, config1: config_a_with_ipBlock, config2: config_b_with_ipBlock, key: Removed connections between persistent peers||||||
||[default]|[app=app-0]|[default]|[app=app-2]|All connections|
|semantic_diff, config1: config_a_with_ipBlock, config2: config_b_with_ipBlock, key: Added connections between persistent peers and ipBlocks||||||
|||0.0.0.0-9.255.255.255|[default]|[app=app-1]|All connections|
|||10.10.0.0/16|[default]|[app=app-1]|All connections|
|||11.0.0.0-255.255.255.255|[default]|[app=app-1]|All connections|
|semantic_diff, config1: config_a_with_ipBlock, config2: config_b_with_ipBlock, key: Removed connections between persistent peers and ipBlocks||||||
|||10.0.0.0-10.10.255.255|[default]|[app=app-2]|All but UDP 53|
|||10.12.0.0-10.255.255.255|[default]|[app=app-2]|All but UDP 53|
|||0.0.0.0-9.255.255.255|[default]|[app=app-2]|All connections|
|||10.11.0.0/16|[default]|[app=app-2]|All connections|
|||11.0.0.0-255.255.255.255|[default]|[app=app-2]|All connections|
|semantic_diff, config1: config_a_with_ipBlock, config2: config_b_with_ipBlock, key: New connections between persistent peers and added peers||||||
||[default]|[app in (app-5,app-6)]|[default]|[app in (app-0,app-1)]|All connections|
||[default]|[app not in (app-5,app-6)]|[default]|[app in (app-5,app-6)]|All connections|
|semantic_diff, config1: config_a_with_ipBlock, config2: config_b_with_ipBlock, key: New connections between added peers||||||
||[default]|[app=app-5]|[default]|[app=app-6]|All connections|
||[default]|[app=app-6]|[default]|[app=app-5]|All connections|
|semantic_diff, config1: config_a_with_ipBlock, config2: config_b_with_ipBlock, key: New connections between added peers and ipBlocks||||||
|||0.0.0.0/0|[default]|[app in (app-5,app-6)]|All connections|
||[default]|[app in (app-5,app-6)]||0.0.0.0/0|All connections|
