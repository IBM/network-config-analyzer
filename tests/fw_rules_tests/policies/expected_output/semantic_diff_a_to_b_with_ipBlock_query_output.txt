config_a_with_ipBlock and config_b_with_ipBlock are not semantically equivalent.

Lost connections between removed peers (based on topology from config: config_a_with_ipBlock) :
src_ns: [default] src_pods: [app=app-3] dst_ns: [default] dst_pods: [app=app-4] conn: All connections
src_ns: [default] src_pods: [app=app-4] dst_ns: [default] dst_pods: [app=app-3] conn: All connections

Lost connections between removed peers and ipBlocks (based on topology from config: config_a_with_ipBlock) :
src: 0.0.0.0/0 dst_ns: [default] dst_pods: [app in (app-3,app-4)] conn: All connections
src_ns: [default] src_pods: [app in (app-3,app-4)] dst: 0.0.0.0/0 conn: All connections

Lost connections between removed peers and persistent peers (based on topology from config: config_a_with_ipBlock) :
src_ns: [default] src_pods: [app in (app-3,app-4)] dst_ns: [default] dst_pods: [app in (app-0,app-2)] conn: All connections
src_ns: [default] src_pods: [app not in (app-3,app-4)] dst_ns: [default] dst_pods: [app in (app-3,app-4)] conn: All connections

Added connections between persistent peers (based on topology from config: config_b_with_ipBlock) :
src_ns: [default] src_pods: [app=app-0] dst_ns: [default] dst_pods: [app=app-1] conn: All connections

Removed connections between persistent peers (based on topology from config: config_a_with_ipBlock) :
src_ns: [default] src_pods: [app=app-0] dst_ns: [default] dst_pods: [app=app-2] conn: All connections

Added connections between persistent peers and ipBlocks (based on topology from config: config_b_with_ipBlock) :
src: 0.0.0.0-9.255.255.255 dst_ns: [default] dst_pods: [app=app-1] conn: All connections
src: 10.10.0.0/16 dst_ns: [default] dst_pods: [app=app-1] conn: All connections
src: 11.0.0.0-255.255.255.255 dst_ns: [default] dst_pods: [app=app-1] conn: All connections

Removed connections between persistent peers and ipBlocks (based on topology from config: config_a_with_ipBlock) :
src: 0.0.0.0-9.255.255.255 dst_ns: [default] dst_pods: [app=app-2] conn: All connections
src: 10.0.0.0-10.10.255.255 dst_ns: [default] dst_pods: [app=app-2] conn: All but UDP 53
src: 10.11.0.0/16 dst_ns: [default] dst_pods: [app=app-2] conn: All connections
src: 10.12.0.0-10.255.255.255 dst_ns: [default] dst_pods: [app=app-2] conn: All but UDP 53
src: 11.0.0.0-255.255.255.255 dst_ns: [default] dst_pods: [app=app-2] conn: All connections

New connections between persistent peers and added peers (based on topology from config: config_b_with_ipBlock) :
src_ns: [default] src_pods: [app in (app-5,app-6)] dst_ns: [default] dst_pods: [app in (app-0,app-1)] conn: All connections
src_ns: [default] src_pods: [app not in (app-5,app-6)] dst_ns: [default] dst_pods: [app in (app-5,app-6)] conn: All connections

New connections between added peers (based on topology from config: config_b_with_ipBlock) :
src_ns: [default] src_pods: [app=app-5] dst_ns: [default] dst_pods: [app=app-6] conn: All connections
src_ns: [default] src_pods: [app=app-6] dst_ns: [default] dst_pods: [app=app-5] conn: All connections

New connections between added peers and ipBlocks (based on topology from config: config_b_with_ipBlock) :
src: 0.0.0.0/0 dst_ns: [default] dst_pods: [app in (app-5,app-6)] conn: All connections
src_ns: [default] src_pods: [app in (app-5,app-6)] dst: 0.0.0.0/0 conn: All connections
